marklogic-client-api/src/main/java/com/marklogic/client/document/DocumentManager.java

```diff
@@ -593,6 +593,19 @@ DocumentDescriptor exists(String docId, Transaction transaction)
    */
   DocumentPage search(QueryDefinition querydef, long start);
 
+  /**
+   * Just like {@link QueryManager#search(QueryDefinition, SearchReadHandle, long) QueryManager.search}
```

Comment:
Should the link both here and in the other new methods go to the equivalent overload of QueryManager.search() that has the same arguments including the forestName?

Reply:
Yes. Changed those links